### ISSUE TITLE
Should eventually add this to the developer documentation

### ISSUE THREAD

**Issue by jkhartshorne** [pushkin-consortium/pushkin#109]
Prep now updates the worker's environment variables in docker-compose.dev.yml based on the database definition in pushkin.yaml, ignoring what is in the experiment's config file.

**Comment by jessestorbeck** [comment]
Covered [here](https://languagelearninglab.gitbook.io/pushkin/advanced/pushkin-cli#details)?

**Comment by jkhartshorne** [comment]
I don't think it is. And this is actually a little confusing. I'm assigning myself to look into it.
